pkgs/by-name/au/authenticator-rs/fix-install.patch

```diff
@@ -0,0 +1,133 @@
+ 	$(INSTALL_DATA) data/icons/hicolor/64x64/apps/uk.co.grumlimited.authenticator-rs.64.png $(sharedir)/icons/hicolor/64x64/apps/uk.co.grumlimited.authenticator-rs.png
+@@ -71,7 +69,7 @@ install : release gresource
+ 	# Force icon cache refresh
+ 	touch $(sharedir)/icons/hicolor
+ 
+-	# Install application meta-data
++	# Install application metadata
+ 	$(INSTALL_DATA) data/uk.co.grumlimited.authenticator-rs.appdata.xml $(sharedir)/metainfo/uk.co.grumlimited.authenticator-rs.appdata.xml
+ 
+ 	# Install desktop file
+@@ -79,12 +77,11 @@ install : release gresource
+ 
+ 	# Install gschema file
+ 	$(INSTALL_DATA) data/uk.co.grumlimited.authenticator-rs.gschema.xml $(sharedir)/glib-2.0/schemas/
+-	
++
+ 	# Install LOCALE files
+ 	rm -fr builddir/
+-	meson builddir --prefix=/usr
++	meson setup --prefix=$(PREFIX) builddir
+ 	DESTDIR=../build-aux/i18n meson install -C builddir
+-	cp -fr build-aux/i18n/usr $(DESTDIR)
+ 
+ # Remove an existing install from the system
+ uninstall :
+@@ -93,7 +90,7 @@ uninstall :
+ 	# Remove the application metadata
+ 	rm -f $(sharedir)/metainfo/uk.co.grumlimited.authenticator-rs.appdata.xml
+ 	# Remove gschema
+-	rm -f /usr/share/glib-2.0/schemas/uk.co.grumlimited.authenticator-rs.gschema.xml
++	rm -f $(sharedir)/glib-2.0/schemas/uk.co.grumlimited.authenticator-rs.gschema.xml
+ 	# Remove the icon
+ 	rm -f $(sharedir)/icons/hicolor/scalable/apps/uk.co.grumlimited.authenticator-rs.svg
+ 	rm -f $(sharedir)/icons/hicolor/64x64/apps/uk.co.grumlimited.authenticator-rs.png
+@@ -102,7 +99,7 @@ uninstall :
+ 	rm -f $(bindir)/bin/authenticator-rs
+ 
+ 	# Remove LOCALE files
+-	find /usr/share/locale/ -name authenticator-rs.mo -exec rm {} \;
++	find $(sharedir)/locale/ -name authenticator-rs.mo -exec rm {} \;
+ 
+ # Remove all files
+ clean-all : clean
+diff --git a/README.md b/README.md
+index bd5b29a..9083dce 100644
+--- a/README.md
++++ b/README.md
+@@ -63,7 +63,7 @@ Download from the [release](https://github.com/grumlimited/authenticator-rs/rele
+ 
+ #### Arch Linux
+ 
+-Authenticator RS is also available from [AUR](https://aur.archlinux.org/packages/authenticator-rs-bin/). Or from the release page:
++Authenticator is also available from [AUR](https://aur.archlinux.org/packages/authenticator-rs-bin/). Or from the release page:
+ 
+     pacman -U authenticator-rs-bin-x.x.x-1-x86_64.pkg.tar.xz
+     
+diff --git a/data/resources/gtk/ui/main.ui b/data/resources/gtk/ui/main.ui
+index 750f2f6..7ae75cf 100644
+--- a/data/resources/gtk/ui/main.ui
++++ b/data/resources/gtk/ui/main.ui
+@@ -906,7 +906,7 @@
+                 <property name="valign">center</property>
+                 <property name="hexpand">True</property>
+                 <property name="vexpand">True</property>
+-                <property name="label">Authenticator RS</property>
++                <property name="label">Authenticator</property>
```

Comment:
IMO this is not necessary. Also it renames the application from what the authors wanted, I would object that.